news/7688.doc

```diff
@@ -0,0 +1 @@
+Add ``-no-input`` option to pip docs
```

Comment:
```suggestion
Add ``--no-input`` option to pip docs
```

Reply:
Fixed in https://github.com/pypa/pip/pull/8182/commits/6119a837ea103276025faaec1feb1f49429271a3